src/enumerable.rb

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+module Enumerable
+  def my_each
+    return to_enum unless block_given?
+
+    length.times { |i| yield(self[i]) }
+    self
```

Comment:
**[REQUIRED]**
Try another way to create your `each` method. Right now it does not work with range. Range does not have length.
`(10..15).each {|n| print n, ' ' }`
`# prints: 10 11 12 13 14 15`